lib/xcake/xcode/project.rb

```diff
@@ -106,6 +106,29 @@ def new_target(target)
           native_target
         end
 
+        # Creates a new xcode configuration from the configuration DSL
+        #
+        # @param [Configurarion] configuration
+        #                 configuration DSL to create target from
+        #
+        # @return [Configurarion] new xcode configuration
+        #
+        def new_configuration(configuration)
```

Comment:
Unused method argument - `configuration`. If it's necessary, use `_` or `_configuration` as an argument name to indicate that it won't be used. You can also write as `new_configuration(*)` if you want the method to accept any arguments but don't care about them.